sxt/multiexp/pippenger2/in_memory_partition_table_accessor_utility.h

```diff
@@ -31,8 +31,12 @@ namespace sxt::mtxpp2 {
 //--------------------------------------------------------------------------------------------------
 // make_in_memory_partition_table_accessor
 //--------------------------------------------------------------------------------------------------
-template <bascrv::element T>
-std::unique_ptr<partition_table_accessor<T>> make_in_memory_partition_table_accessor(
+template <class U, bascrv::element T>
```

Comment:
Is there a reason some of the templates are defined by `template <bascrv::element T, class U>` and others by `template <class U, bascrv::element T>`? I wonder if that will lead to to incorrect ordering of template parameters?

Reply:
It's possible to deduce the T type here, but the U type needs to be specified; that's why it's first.

I don't think there's a type mismatch anywhere. In general, type ordering can be different for different functions.